From b921763db4fd56ef4890f1606fe9e6a1841780aa Mon Sep 17 00:00:00 2001
From: mliang2x <mingyuex.liang@intel.com>
Date: Wed, 26 Aug 2020 09:38:05 +0800
Subject: [PATCH] Test case2 for Change build option under BuildOptions.IA32

Description: Change build option under BuildOptions.IA32 ("/D UNICODE") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..8c788f0fae 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -33,7 +33,7 @@
   CLANGPDB:*_*_*_DLINK_FLAGS = /ALIGN:4096
   
 [BuildOptions.IA32]
-  RELEASE_MYTOOLS_IA32_CC_FLAGS        = /D MDEPKG_NDEBUG
+  RELEASE_MYTOOLS_IA32_CC_FLAGS        = /D UNICODE
 
 [BuildOptions.IA32.EDKII.PEIM]
   GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000
-- 
2.25.1.windows.1

